src/app/guest-book/entries-list/entries-list.component.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <div *ngIf="!(entries$ | async)">
+    <mat-progress-bar mode="query"></mat-progress-bar>
+  </div>
+  <div class="entries-grid" *ngIf="entries$ | async as entries">
```

Comment:
you can change 2nd and 5th line - and use <div *ngIf="!entries"> in 2nd line

Reply:
it seems I misinterpret the suggestion as switching progress bar and entries-grid places doesn't work out
could you please elaborate slightly, if I may ask?